pyopencl/__init__.py

```diff
@@ -1412,6 +1384,25 @@ def gl_object_get_gl_object(self):
 
 # }}}
 
+# {{{ _ArrayInterfaceSVMAllocation
+
+if get_cl_header_version() >= (2, 0):
+    class _ArrayInterfaceSVMAllocation(SVMAllocation):
+        def __init__(self, ctx, size, alignment, flags, _interface=None,
+                queue=None):
+            """
+            :arg ctx: a :class:`Context`
+            :arg flags: some of :class:`svm_mem_flags`.
+            """
+            super().__init__(ctx, size, alignment, flags, queue)
+
+            # mem_flags.READ_ONLY applies to kernels, not the host
+            read_write = True
+            _interface["data"] = (
+                    int(self.int_ptr), not read_write)
+
```

Comment:
This fixes `examples/svm.py` for me:

```suggestion

self.__array_interface__ = _interface
```

Reply:
Thanks, done! (3049df0)